Utils/Elasticsearch/requests/hashtag-steps-statistics-q.json

```diff
@@ -38,9 +36,21 @@ GET <index>/task/_search
         "cpu": {
           "sum": {"field": "hs06"}
         },
-        "walltime": {
-         "sum": {"script": {"inline":"doc['end_time'].value - doc['start_time'].value"}}
-        },
+		"timestamp_defined": {
+			"filter": {
+             "bool": {
+               "must": [
+                 {"exists": {"field": "start_time"}},
+                 {"exists": {"field": "end_time"}}
+               ]
+             }
+           },
+          "aggs": {
+            "input_bytes": {
+              "sum": {"script": {"inline":"doc['end_time'].value - doc['start_time'].value"}}
```

Comment:
And the last: please, add missed space after colon: `"inline":"doc['end_time']...`